README.md

```diff
@@ -120,16 +120,14 @@ systemctl enable betterlockscreen@$USER
 
 ## Configuration
 
-You can customize betterlockscreen for your needs, copy the config file from the examples-directory to `~/.config/betterlockscreenrc` and edit it accordingly.
-
-If no configuration-file is found, then the default configurations (which is equal to the example but currently hardcoded) will be used.
-
-If you have installed betterlockscreen from AUR package, then you can copy default config from docs
```

Comment:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=betterlockscreen#n23
Does not apply anymore.